manticore/platforms/linux.py

```diff
@@ -3306,6 +3338,21 @@ def _interp_total_size(interp):
         last = load_segs[-1]
         return last.header.p_vaddr + last.header.p_memsz
 
+    @staticmethod
+    def implemented_syscalls() -> Iterable[str]:
+        import inspect
+
+        return (
+            x[0].split("sys_", 1)[1]
+            for x in inspect.getmembers(Linux, predicate=inspect.isfunction)
+            if x[0].startswith("sys_")
+        )
+
+    @staticmethod
+    def print_implemented_syscalls() -> None:
+        for syscall in Linux.implemented_syscalls():
+            print(syscall)
```

Comment:
These are some helpful methods for providing more info about Manticore's syscall implementation state

Reply:
Opened a PR https://github.com/trailofbits/manticore/pull/2491

I didn't include the `print_implemented_syscalls` method because it's very easily implemented by the user and probably doesn't belong in the library